docs/installation.md

```diff
@@ -14,12 +27,14 @@ sudo mv ./gitops /usr/local/bin/gitops
 gitops version
 ```
 
-You should see:
+you should see:
 
 ```console
 Current Version: v0.0.0
 GitCommit: c0d130b
-BuildTime: 2021-11-03_20:00:24
+
+##BuildTime: 2021-11-03_20:00:24
 Branch: HEAD
-Flux Version: v0.21.0
+### Kubernetes Cluster
+Weave GitOps is compatable with conformant Kubernetes distributions which match the minimum required version level of Flux x Version: v0.21.0
```

Comment:
Same as above

Reply:
hmm, this looks like weird duplication from the intro, remove and resolved